src/include/migraphx/onnx.hpp

```diff
@@ -53,6 +53,10 @@ struct onnx_options
     bool print_program_on_error = false;
     /// Max iter num for the loop operator
     int64_t max_loop_iterations = 10;
+    /*!
+     * Set the use_dyn_output attribute to true for all pertinent operators
+     */
```

Comment:
Use `///` like the rest of the comments(The comments for dynamic dimensions should probably be updated as well to match). And I would write something like `/// Use dynamic output for operators when available`, you dont need to mention an attribute as that is an implementation detail(we could use different operators instead of an attribute).